### ISSUE TITLE
Sponsoring guidelines/best practices for meetups

### ISSUE THREAD

**Issue by campoy** [golang/cwg#35]
How do meetups get sponsoring?

I think the organizers from successful meetups could give their best practices for those that are starting.

**Comment by corylanou** [comment]
from here: https://github.com/corylanou/go-meetup#how-do-i-get-sponsors


### How do I get sponsors?
> This can take a little more work, but the first stop is your current employer! Even $100 is a great start to buy a couple pizzas and some refreshments. Be sure to approach any company that currently has a Go project or is hiring Go developers. I've also found that any company that does recruiting usually has a budget to support meetups. I'm hoping to work on a project next year that allows us to have a world wide fund for meetups that are getting started, but I can't make any assurance that endeavor will be successful.

**Comment by corylanou** [comment]
Also, fwiw, I find that most of the time the venue that is hosting you will likely sponsor the food/drinks as well.  When Go was new it was VERY hard to get sponsors.  I find getting sponsors now to be not much of a problem after you show them you will consistently host a meetup.

**Comment by danicat** [comment]
I second everything @corylanou said. I'm not sure if I qualify as a `successful` meetup organizer since I've haven't organized many events so far, but even though I'm just beginning I've been already approached by several parties interesting in sponsoring our meetups.

Most of my sponsors are actively recruiting and the investment is pretty small compared to the positive publicity it generates. I usually ask for the location and a coffee break if the event runs for less than 4 hours.

When the event lasts the entire day the costs are a bit higher because I ask for t-shirts and extra meals, along with any gifts the company is able to offer: pens, stickers, bags, etc.

Finally, I had better experience with small to mid-size companies. The big companies usually have a larger budget but the competition is higher too.

**Comment by arjanvaneersel** [comment]
I agree with the remarks of @corylanou. Regarding Bulgaria I can say that the general IT community in Sofia is quite vivid and opened to new initiatives. And Go is booming in Sofia, we're now having 133 member in the meetup and the average amount of meetup visitors is around 40. There are various companies offering us a space which we can use for free, so that seriously cuts the costs. One company usually sponsors some pizza's. 

Also we sometimes have "outdoor trips", which offer a look in the kitchen of companies using Go. The 2nd meetup we held was at Uber, which has a big development office in Sofia. This was a very popular meetup and didn't cost us anything.

I'm now looking into the option of having meetups in Plovdiv as well. Although Plovdiv is also a big city, the IT community is more modest than in Sofia and Go is also less known there, so far haven't been able to find any companies using Go in that city and therefore things are a bit more difficult than in Sofia. I'm in touch with a Hackerspace there and they are very positive about offering their location for the meetings. For them it's a new activity they can offer to their members, for the meetup it's a free location, so everybody wins. So I'd definitely would recommend to find strategic partners, like hacker spaces, coworking facilities, etc.

**Comment by nathany** [comment]
For food and drinks, the organizers have been covering that cost for the 4 years we've been running it, though our meetup is relatively small at 20ish people. We have a government-funded co-op space that graciously hosts our meetup along with many others. 

We've also paid for and raffled off prizes such as books and gopher swag. We've done one paid workshop and used the money to provide attendees with swag as well as pay for the speaker's airfare. We've also done one free workshop for university students.

Just saw this from Cloudflare the other day but haven't looked into it yet:
https://blog.cloudflare.com/cloudflare-wants-to-buy-your-meetup-group-pizza/
